src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -230,10 +232,10 @@ protected virtual Task RequestExecutedAsync(in GraphQLRequestExecutionResult req
                 }
             };
 
-            httpResponse.ContentType = "application/json";
-            httpResponse.StatusCode = (int)httpStatusCode;
+            context.Response.ContentType = "application/json";
+            context.Response.StatusCode = (int)httpStatusCode;
 
-            return serializer.WriteAsync(httpResponse.Body, result, cancellationToken);
+            return _serializer.WriteAsync(context.Response.Body, result, context.RequestAborted);
```

Comment:
cancellationToken is lost, see ` var cancellationToken = GetCancellationToken(context);` line

Reply:
That method should not exist.  For now I changed the code to use it.

Reply:
Why should not? I use it to link requestaborted token with my custom token to handle user defined operation timeout from request headers.

Reply:
Also see ` protected override CancellationToken GraphQLHttpMiddlewareWithLogs.GetCancellationToken(HttpContext context)`